### PR TITLE
feat: update glossary term definitions [FIXES #12365]

### LINKED ISSUE #12365: Update glossary terms
Update these terms to our new glossary JSON structure  from old PR, ignore others as they have been updated in a different ticket. 

ABI
block validation
bootnode
bytecode
committee
consensus client
finality
fork
fork-choice algorithm
fraud proof
Merkle Patricia trie
ommer (uncle) block
proof-of-work
reward
Recursive Length Prefix (RLP)
shard / shard chain
terminal total difficulty (TTD)

from this old PR: https://github.com/ethereum/ethereum-org-website/pull/10975

### PR DESCRIPTION
**Pull request by tomasgrusz**
- update requested term definitions in `glossary.json`
- update requested term definitions in `glossary-tooltip.json`

## Description

This PR contains updates to the term definitions in the glossary that were introduced in [#10975](https://github.com/ethereum/ethereum-org-website/pull/10975), which was closed, but requested from @konopkja in [#12365](https://github.com/ethereum/ethereum-org-website/pull/12365) to be implemented again in the new glossary structure.

Terms in question:
> ABI, block validation, bootnode, bytecode, committee, consensus client, finality, fork, fork-choice algorithm, fraud proof, Merkle Patricia tree, ommer (uncle) block, proof-of-work, reward, Recursive Length Prefix (RLP), shard / shard chain, terminal total difficulty (TTD)
>
> _NOTE: the term `committee` already contains the requested definition, therefore nothing was changed_

## Related Issue
[Update glossary terms #12365](https://github.com/ethereum/ethereum-org-website/issues/12365)

### CODE REVIEW

**Comment by konopkja** [comment]
lgtm, thank you for the contribution! <3 Once dev will review the change (this may take few days), we will merge it to production!

File: src/intl/en/glossary.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
...
   "consensus-client-term": "Consensus client",
-  "consensus-client-definition": "Consensus clients (such as Prysm, Teku, Nimbus, Lighthouse, Lodestar) run Ethereum's <a href=\"/glossary/#pos\">proof-of-stake</a> consensus algorithm allowing the network to reach agreement about the head of the Beacon Chain. Consensus clients do not participate in validating/broadcasting transactions or executing state transitions. This is done by <a href=\"/glossary/#execution-client\">execution clients</a>.",
+  "consensus-client-definition": "Consensus clients (such as Prysm, Teku, Nimbus, Lighthouse, Lodestar) run Ethereum's <a href=\"/glossary/#pos\">proof-of-stake</a> consensus algorithm allowing the network to reach agreement about the head of the Beacon Chain. Consensus clients do not participate in validating/broadcasting transactions or executing state transitions. This is done by <a href=\"/glossary/#execution-client\">execution clients</a>. Consensus client do not attest to, or propose new blocks. This is done by the <a href=\"/glossary/#validator\">validator client</a> which is an optional add-on to the consensus client.",
```
Comment (wackerow) [inline src/intl/en/glossary.json:69]:
```suggestion
  "consensus-client-definition": "Consensus clients (such as Prysm, Teku, Nimbus, Lighthouse, Lodestar) run Ethereum's <a href=\"/glossary/#pos\">proof-of-stake</a> consensus algorithm allowing the network to reach agreement about the head of the Beacon Chain. Consensus clients do not participate in validating/broadcasting transactions or executing state transitions. This is done by <a href=\"/glossary/#execution-client\">execution clients</a>. Consensus clients do not attest to, or propose new blocks. This is done by the <a href=\"/glossary/#validator\">validator client</a> which is an optional add-on to the consensus client.",
```

**Review by wackerow — APPROVED** [review]
Lgtm! Just two small patches that I'll bring in, thanks @tomasgrusz!

**Comment by wackerow** [comment]
@all-contributors please add @tomasgrusz for code

**Pull request merged**